plugins/polio/js/src/hooks/useGetCampaigns.js

```diff
@@ -69,16 +67,21 @@ export const useGetCampaigns = (
     // adding the params to the queryKey to make sure it fetches when the query changes
     return {
         // eslint-disable-next-line no-return-assign
-        exportToCSV: () =>
-            (window.location.href = `${getURL({
-                ...params,
-                limit: undefined,
-                page: undefined,
-                format: 'csv',
-            })}`),
+        exportToCSV: `${getURL({
+            ...params,
+            limit: undefined,
+            page: undefined,
+            format: 'csv',
```

Comment:
wouldn't it be more consistent to just override the fieldset here like we do for the page and limit?